src/main/java/seedu/address/logic/commands/ImportExamScoresCommand.java

```diff
@@ -37,17 +30,21 @@ public class ImportExamCommand extends Command {
     public static final String MESSAGE_SCORE_NOT_NUMBER = "Score for %s is not a number";
     public static final String MESSAGE_PERSON_DOES_NOT_EXIST = "Person does not exist";
     public static final String MESSAGE_SUCCESS = "Imported exams from: %s";
-    public static final String PREFIX_SCORE = " 1 s/";
-    public static final String PREFIX_FIND_BY_EMAIL = " e/";
+    public static final String MESSAGE_DUPLICATE_EXAM =
+            "Duplicate exam header. Only the records of the first exam will be imported if present.";
+    public static final String EXAM_HEADER = "Exam:";
```

Comment:
Nice job on adding the exam header. Now we can explicitly import exams instead of having ambiguities in our csv import functions